src/style.css

```diff
@@ -1 +1,74 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.form__search-big {
+  font-family: Avenir-Book, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 21.86px;
+  color: rgba(61, 78, 97, 1);
+  width: 100%;
+  height: 70px;
+  outline: none;
+  margin-top: 12px;
```

Comment:
The margin-top for the big search bar should be 20px to meet the task requirement of having a top indent of 20px.